services/core-web/src/components/Forms/noticeOfWork/EditNoWContacts.js

```diff
@@ -152,7 +151,7 @@ const NOWContact = ({
               style={{ fontSize: "1.5rem", fontWeight: "bold" }}
               to={router.PARTY_PROFILE.dynamicRoute(contact.party.party_guid)}
             >
-              {formatTitleString(contact.party.name)}
+              {contact.party.name}
```

Comment:
I have a feeling we shouldn't really be using this function anywhere? Should we remove other instances of it being used in the code?

Reply:
I removed it everywhere.

Reply:
> Can we also make the casing consistent for the form fields in this screenshot?
> ![image](https://user-images.githubusercontent.com/17113094/117179561-1ab29200-ad88-11eb-8432-5437b1e32e28.png)
> Each word should start with an uppercase letter

Fixed.